src/test/java/se/iths/crimedatabase/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,106 @@
+package se.iths.crimedatabase.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.iths.crimedatabase.entity.Category;
+import se.iths.crimedatabase.repository.CategoryRepository;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class CategoryServiceTest {
+
+    @Mock
+    private CategoryRepository repository;
+    @InjectMocks
+    private CategoryService service;
+
+    // CREATE
+    // When we call the create-method with a category,
+    // said category should be passes to the save method of the repository
+    @Test
+    void callingCreateShouldReturnTheCreatedCategory() {
+        Category category = new Category();
+        category.setName("Test").setId(1L);
+        when(repository.save(category)).thenReturn(category);
+
+        Category createdCategory = service.create(category);
+
+        assertThat(createdCategory).isEqualTo(category);
+    }
+
+    // FIND ALL
+    // When we call the findAll-method a List of all categories should be returned.
+    @Test
+    void callingFindAllShouldReturnAllCategories() {
+        List<Category> categories = List.of(
+                new Category().setName("Theft").setId(1L),
+                new Category().setName("Burglary").setId(2L),
+                new Category().setName("shoplifting").setId(3L));
```

Comment:
```suggestion
                new Category().setName("Shoplifting").setId(3L));
```
Let's change to a capital 'S' to correspond with the other categories and look better